WordPress/Classes/ViewRelated/System/WPTabBarController.h

```diff
@@ -21,27 +23,19 @@ extern NSNotificationName const WPTabBarHeightChangedNotification;
 
 @interface WPTabBarController : UITabBarController <UIViewControllerTransitioningDelegate>
 
-@property (nonatomic, strong, readonly) NotificationsViewController *notificationsViewController;
-@property (nonatomic, strong, readonly) UINavigationController *readerNavigationController;
-@property (nonatomic, strong, readonly) MySitesCoordinator *mySitesCoordinator;
-@property (nonatomic, strong, readonly) ReaderCoordinator *readerCoordinator;
-@property (nonatomic, strong, readonly) BloggingPromptCoordinator *bloggingPromptCoordinator;
+@property (nonatomic, strong, readonly, nullable) NotificationsViewController *notificationsViewController;
+@property (nonatomic, strong, readonly, nullable) UINavigationController *readerNavigationController;
+@property (nonatomic, strong, readonly, nullable) MySitesCoordinator *mySitesCoordinator;
```

Comment:
By looking at the implementation of `RootViewPresenter` in  `WPTabBarController.m` and `MySitesCoordinator` I think it could be `nonnull.` 

```suggestion
@property (nonatomic, strong, readonly, nonnull) MySitesCoordinator *mySitesCoordinator;
```

Reply:
I agree. Updated in 1af613bc6594f9fd0b845188dd597a727ec6c73c 👍